USAGE.md

```diff
@@ -0,0 +1,105 @@
+  See [Item Definition and Syntax](http://docs.openhab.org/configuration/items.html#item-definition-and-syntax)
+  
+   ```java
+   Switch Kitchen_Light "Kitchen Light" <light> (gKitchen) [ "Lighting" ] {channel="..."}
+   ```
+   
+* **Items via PaperUI**
+ 
+   There (actually) is no easy way to tag yout Items via PaperUI.
+ 
+   Some users managed to tag items via REST API.
+   Have a loot at [this post](https://community.openhab.org/t/apply-tags-to-items-added-linked-in-paper-ui/19443/11?u=mboremski).
+   <!--- Are there more relevant ways to configure items? --->
+   <!--- Should we add a chapter for availabletags? --->
+
+### 2. Skill Configuration
+
+1. Visit the [Alexa-Website](https://alexa.amazon.com/) or use the belonging app on your mobile.
+2. Navigate to "Smart Home" -> "Configure Smart Home" and search for openHAB.
+3. Follow the Instructions there to authorise Alexa using your myopenhab.org-account.
+
+### 3. Final Configuration
+
+After tagging your items you can go back to the Alexa-configurations and search for devices.
+
+### Example Voice Commands
+
+Here are some example voice commands:
+
+ * "Alexa. turn on Kitchen Light"
+ * "Alexa, turn off Kitchen Light"
+ * "Alexa, turn on Bedroom Light"
+ * "Alexa, turn on Bedroom Light"
+ * "Alexa, dim Kitchen Lights to 30 percent"
+
+### Additional Comments
+
+<!--- you have better suggestions for the Headline? --->
+
+* Thermostats are created by adding the items of a thermostat to a group which has the tag "Thermostat" which follows the HomeKit binding configuration. 
+See [HomeKit Add-on](http://docs.openhab.org/addons/io/homekit/readme.html) for more information on how to configure thermostats.
+Thermostats can have their target temperature set as well as be asked what the current temperature is.
+* Channels which are tagged "CurrentTemperature" but NOT part of a thermostat group will be exposed as a Temperature item in Alexa and can be asked what their current value is ("Alex what is the upstairs temperature? ")
+* By default all temperatures are in Celsius, for Fahrenheit add the tag `Fahrenheit` to the thermostat group item (which should also be tagged with `Thermostat`).
+For standalone temperature channels, add it directly to the item.
+* In addition you can tag Rollershutter items by `[ "Switchable" ]` and get support for `setPercentage`, `incrementPercentage`and `decrementPercentage` commands.
+Example:
+  ```java
+  Rollershutter Shutter_GF_Kitchen "Rollershutter Kitchen" [ "Switchable" ]
+  ```
+* With commands like `Alexa, set rollershutter kitchen to 100%` you control the rollershutter in the kitchen.
+* If your rollershutters or blinds happen not to support aperture by percentage the following rule helps to achieve opening and closing:
+  ```java
+  rule Rollershutter_Kitchen
+  when
+      Item Shutter_GF_Kitchen received command
+  then
+      if (receivedCommand < 50) { // in germany alexa often recognizes "0" as "9"
```

Comment:
The comment is misplaced and irrelevant, probably untrue by now...

Reply:
You want me to remove whole chapter?

Reply:
Will remove rule as mentioned by @digitaldan